src/autoscaler/cf/app_test.go

```diff
@@ -17,6 +18,12 @@ import (
 	"net/url"
 )
 
+type ErrorResponse struct {
```

Comment:
you may want to move this to https://github.com/cloudfoundry-incubator/app-autoscaler/blob/develop/src/autoscaler/models/errors.go